service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/BaseDocmosisDocument.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.fpl.model.docmosis;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.Map;
+
+abstract class BaseDocmosisDocument {
```

Comment:
can you rename this AbstractDocmosisData so I don't have headaches refactoring 😁